src/libraries/SwapMath.sol

```diff
@@ -42,7 +42,8 @@ library SwapMath {
     /// @return sqrtPriceNextX96 The price after swapping the amount in/out, not to exceed the price target
     /// @return amountIn The amount to be swapped in, of either currency0 or currency1, based on the direction of the swap
     /// @return amountOut The amount to be received, of either currency0 or currency1, based on the direction of the swap
-    /// @return feeAmount The amount of input that will be taken as a fee
+    /// @return feeAmount The amount of input that will be taken as a fee/
+    /// @dev feePips must be no larger than MAX_FEE_PIPS
```

Comment:
It seems like 39 is saying its not guaranteed to be less than MAX_FEE_PIPS. Can we make this comment say it is guranteed because feePips is capped before computeSwapStep? 

And then I think we need a fuzz test to ensure that a) calculateSwapFee < MAX and b) setLpFee never allows setting greater than MAX

Reply:
We already have the following tests:
- `test_fuzz_calculateSwapFee` ensures that the swap fee never exceeds max
- `test_updateDynamicLPFee_beforeSwap_failsWithTooLargeFee` checks that updateDynamicLPFee never allows a fee>max
- `test_fuzz_dynamicReturnSwapFee` checks that the beforeSwap hook's returned dynamic fee cannot be too large
- `test_fuzz_initialize` checks that no non-dynamic fee pool can be initialized with a fee larger than max
- `test_fuzz_getInitialLPFee` checks that the initial LP fee for a pool is never larger than max

Reply:
i've added the comment to make it clearer, but let me know if theres another test you think is missing and i can code one!

Reply:
perfect! those tests sound like they cover this

Reply:
& comment looks good